README.md

```diff
@@ -1,25 +1,25 @@
 
-But first, we have to acknowledge our company's role in our warming planet. 
+Our company should reconsider its technology support of oil and gas exploitation, and should have a deliberate plan to withdraw our innovation from that sector, as a matter of principal.
 
-Microsoft makes millions of dollars in profits by helping fossil fuel companies extract more oil. In this week's company-wide all-hands, our CEO, Satya Nadella, defended our contracts with Big Oil by suggesting that our technology helps them in their R&D efforts to develop sustainable energy. Yet, this is in clear contradiction to the ambitions of fossil fuel-CEOs and even other Microsoft leaders: Azure Executive Tom Keane, in discussing [Microsoft's multimillion dollar contract with Chevron](https://news.microsoft.com/transform/chevron-fuels-digital-transformation-with-new-microsoft-partnership/), asked "How can we [..] more efficiently do oil exploration? Or more efficiently do sensor management from an offshore oil rig?" 
+One could defend our contracts with oil and gas by suggesting that our technology helps in their R&D efforts to develop sustainable energy. Yet, Azure Executive Tom Keane, in discussing [Microsoft's multimillion dollar contract with Chevron](https://news.microsoft.com/transform/chevron-fuels-digital-transformation-with-new-microsoft-partnership/), asked "How can we [..] more efficiently do oil exploration? Or more efficiently do sensor management from an offshore oil rig?" 
 
 Furthermore, Microsoft struck a deal last year with Norweigen petroleum company Equinor worth ["hundreds of millions of dollars"](https://www.marketscreener.com/EQUINOR-1413290/news/Equinor-and-Microsoft-in-Cloud-Services-Partnership-Deal-26794775/) and participated in an oil and gas conference with the goal of ["Empowering Oil and Gas with AI."](https://news.microsoft.com/en-xm/2018/11/12/microsoft-demonstrates-the-power-of-ai-and-cloud-to-oil-and-gas-players-at-adipec-2018/ ) Earlier this year, Microsoft announced a new contract with [Exxon Mobil](https://www.reuters.com/article/us-exxon-mobil-microsoft-cloud/exxon-microsoft-strike-cloud-computing-agreement-for-u-s-shale-idUSKCN1QB1N8
-), which the company boasts to be "the industry's largest in cloud computing." 
+), which the company says is "the industry's largest in cloud computing." 
 
-As Microsoft workers, we've been made complicit.
+As Microsoft workers, whilst we are proud of the efforts the company has made towards improving sustainability, we must do more.
```

Comment:
This removes the importance of individual accountability, which is important, and I think we should leave the original wording in place.

Reply:
How do you feel about the guilty by association term 'complicit'?

Reply:
I think the term 'complicit' is the exact right phrase to use here, because we are guilty by association.